dist/index.html

```diff
@@ -99,8 +99,85 @@
         </div>
       </section>
 
-      <section class="slider">
-        Navbar And Slider
+      <section
+        class="slider pt-0 bg-cover bg-center h-screen relative bg-c800 relative"
+        style="
+          background-image: url('assets/img/header-slider/slider.jpg');
+          height: 80vh;
```

Comment:
- You need to add the appropriate values for` background-position `` and background-size`
   so that the image covers the whole area of the targeted element.
-  Instead of using `height:80vh`. you need to change it with the appropriate height to match the design. I think `700px` or `600px` will look better